frontend/src/components/LandingPage/Features/Features.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Row, Col, CardText, CardHeader,Card } from 'reactstrap';
+import { featuresArray }  from '../../../global';
+
+function Features() {
+    return (<div>
```

Comment:
Move `text-center` className from `h1`, `h6`, and `Row` to the top div. This will make the code more concise.

Reply:
Great feedback Dylan. I made the change.